pages/platform/private-registry/configure-oidc-provider-authentication/guide.en-au.md

```diff
@@ -0,0 +1,288 @@
+    groupsClaim: <group in keycloack>
+    name: keycloack //for example
+    scope: openid,profile,email,offline_access
+    userClaim:
+    verifyCert: true //if you want to check the keycloack SSL/TLS certificate
+```
+
+**Result:**
+
+```json
+null
+```
+
+Go to the [Harbor documentation](https://goharbor.io/docs/2.8.0/administration/configure-authentication/oidc-auth/) to see the details of the needed information.
+
+> [!primary]
+>
+> If you forget to enable the `deleteUsers` option, you will have an error: `{ "class": "Client::BadRequest", "message": "4 users exist, set parameter deleteUsers to delete them" }`. Indeed, when a new Private Registry is created, several users are created. So you need to enable the `deleteUsers` option in order to remove existing local users and enable OIDC authentication.
+
+Now, access the Harbor HMI. New buttons should appear, especially the `Login via OIDC provider`{.action}:
+
+![Harbor login via oidc provider](images/login.png)
+
+When you click the `Login via OIDC provider`{.action} button, you are redirected to the Keycloack interface. 
+
+![keycloack login](images/keycloack-login.png)
+
+If you don't enable the `autoOnboard` input field, a popup in the Harbor interface should appear asking you to fill the username.
+When this option is checked, the attribute `UserClaim` must be set, Harbor will read the value of this claim from the ID token and use it as the username for onboarding the user.
+
+![Harbor login via OIDC provider](images/autoonboard_off.png)
+
+Click on the `Save`{.action} button to finally log in to Harbor with the OIDC provider.
+
+- Get an existing registry's OIDC configuration:
+
+> [!api]
+>
+> @api {GET} /cloud/project/{serviceName}/containerRegistry/{registryID}/openIdConnect
+>
+
+**Input:**
+
+```
+serviceName: <your service name>
+registryId: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+```
+
+**Result:**
+
+```json
+{
+  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+  "name": "keycloak",
+  "createdAt": "2023-08-24T13:35:07.217448Z",
+  "updatedAt": "2023-08-24T13:35:07.300496Z",
+  "status": "READY",
+  "endpoint": "https://xxxx.yyy.com/realms/mpr",
+  "clientId": "harbor",
+  "groupsClaim": "groups",
+  "adminGroup": "harbor-admin",
+  "scope": "openid,profile,email,offline_access",
+  "verifyCert": true,
+  "autoOnboard": false,
+  "userClaim": ""
+}
+```
+
+- Update an existing registry's OIDC provider (to add adminGroup and groupsClaim fields for example):
+
+First, retrieve information about groups in keycloack:
+
+![keycloack group](images/keycloack_groups.png)
+
+Check you have at least one member in the group:
+
+![keycloack group member](images/keycloack_group_member.png)
+
+Then, update the configuration with a PUT request.
+
+> [!api]
+>
+> @api {PUT} /cloud/project/{serviceName}/containerRegistry/{registryID}/openIdConnect
+>
+
+```
+OIDCPost:
+  provider:
+    adminGroup: harbor-admin //for example, it's my admin group in keycloack
```

Comment:
adminGroup has already been set at OIDC configuration, maybe we should remove it from there ?